### DISCUSSION TITLE
HPE Books

### DISCUSSION THREAD

**Discussion by sc211166** [apprenticeharper/DeDRM_tools#1970] [Q&A]
I tried to to use the plugin with calibre to import books from HPE bookstore  that I have in my ADE application but it fails.

**Comment by ElleKayEm** [comment]
You have ADE 2.0.1 authorized with your Adobe ID and on the same machine and user as calibre with DeDRM?

**Comment by sc211166** [comment]
I have ADE 4.5 with books authorized on the same machine

**Comment by ElleKayEm** [comment]
Could be hardened DRM which can't be removed.  Try the newer version of DeDRM available here:  https://github.com/noDRM/DeDRM_tools
